common/flatpak-dir.c

```diff
@@ -738,6 +738,13 @@ is_good_installation_id (const char *id)
       strcmp (id, "system") == 0)
     return FALSE;
 
+  if (!g_str_is_ascii (id) ||
```

Comment:
Might people who use non-ASCII languages want to have an installation name in their own language? Or are we assuming not because the rest of the flatpak CLI (command names, options, etc.) are in English?

Reply:
We use the installation name in filenames and keyfile group names in various places, plus, there is a separate DisplayName field defined in flatpak-installation(5), which would presumably be where you let Unicode run wild.

Reply:
Next step would be to ensure that we actually use DisplayName when we present installations to the user. The current FlatpakInstallation api for names/ids is less than ideal atm.